packages/@tailwindcss-upgrade/src/codemods/format-nodes.ts

```diff
@@ -18,7 +18,15 @@ export function formatNodes(): Plugin {
     // Format the nodes
     await Promise.all(
       nodesToFormat.map(async (node) => {
-        node.replaceWith(parse(await format(node.toString(), { parser: 'css', semi: true })))
+        node.replaceWith(
+          parse(
+            await format(node.toString(), {
+              parser: 'css',
+              semi: true,
+              singleQuote: true,
```

Comment:
I noticed that we inconsistently used single quotes for the `@plugin` and `@config` directives but double-quotes for CSS values so this is just a consistency change.